S2VX.Web/wwwroot/css/app.css

```diff
@@ -58,3 +58,15 @@ a, .btn-link {
     transform: translate3d(-50%, -50%, 0);
     filter: brightness(50%);
 }
+
+#blazor-error-ui {
+    background: lightyellow;
+    bottom: 0;
+    box-shadow: 0 -1px 2px rgba(0, 0, 0, 0.2);
+    display: none;
+    left: 0;
+    padding: 0.6rem 1.25rem 0.7rem 1.25rem;
+    position: fixed;
+    width: 100%;
+    z-index: 1000;
+}
```

Comment:
This doesn't solve the problem. Scroll down and you'll still see the text there:
![image](https://user-images.githubusercontent.com/3955187/137652339-134bc83f-80e7-42db-ae6f-6dfe4bff8150.png)

I did some quick testing, and you can remove the `blazor-error-ui` div from `S2VX.Web/wwwroot/index.html`

Reply:
No, you should remove both the div and css sections

Reply:
I removed both in the latest change but it still gives error 😹